mmv1/products/datastream/Stream.yaml

```diff
@@ -1002,6 +1019,26 @@ properties:
             default_from_api: true
             validation: !ruby/object:Provider::Terraform::Validation
               function: 'validation.IntAtLeast(0)'
+          - !ruby/object:Api::Type::NestedObject
+            name: 'transactionLogs'
+            allow_empty_object: true
+            send_empty_value: true
+            exactly_one_of:
```

Comment:
for the most part this all looks good. my main concern is the use of `exactly_one_of` since this would mark both fields as required but from one. I believe we would want to use conflicts instead since the resource itself can be used without setting either `transactionLogs` or `changeTables`.

[Related Documentation on `exactly_one_of`](https://googlecloudplatform.github.io/magic-modules/develop/field-reference/#exactly_one_of)

Reply:
Good call. Switched it up c24b5028c430f371a991dfca4fe308947f264839